PiggyBank.css

```diff
@@ -0,0 +1,85 @@
+* {
+  box-sizing: border-box;
+}
+
+#piggyImg{
```

Comment:
So I noticed you use id's alot.
I was taught to use classes often, and id's less so. It helps with code reuse and specificity issues.

Well done, looks good!

Reply:
Got it!

Reply:
Left a similar comment inside the HTML document.  This comment didn't exist when I started reviewing.  Still not sure how that is supposed to be handled.